CensusPopulationByState.r

```diff
@@ -0,0 +1,21 @@
+install.packages("tidycensus")
+install.packages("tidyverse")
+install.packages("ggplot2")
+
+
+library(tidycensus)
+library(tidyverse)
+library(ggplot2)
+
+census_api_key("6f46a7532e0317f2318ad51b2aa17a2643edcc72", install = TRUE, overwrite=TRUE)
```

Comment:
Don't put your api key in your code. Please see comment in previous review. Research how to store keys as SYSTEM variables, and then call those in your code. You don't want other users using your API keys, even if they're "free" like the census one is.